core/src/main/java/dk/alexandra/fresco/framework/PerformanceLogger.java

```diff
@@ -0,0 +1,44 @@
+package dk.alexandra.fresco.framework;
+
+import java.text.DecimalFormat;
+import java.util.EnumSet;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class for getting performance numbers. Use the EnumSet to indicate which parameters to log. Note
+ * that network logging is done on ONLY the data received and used within FRESCO. This means that if
+ * the network implementation uses double the bytes to wrap the messages, this will not show.
+ * 
+ * @author Kasper Damgaard
+ *
+ */
+public abstract class PerformanceLogger {
```

Comment:
From previous comment - unaddressed:
The inheritance is currently used for passing the partyid into the logging. Change the PerformanceLogger to an interface and let print take the party as an argument? Or a decorated logger. Inheritance is overkill here